src/index.ts

```diff
@@ -34,19 +34,27 @@ export default function useSound(
 
   // We want to lazy-load Howler, since sounds can't play on load anyway.
   useOnMount(() => {
-    import('howler').then(mod => {
-      HowlConstructor.current = mod.Howl;
-
-      const sound = new HowlConstructor.current({
-        src: [url],
-        volume,
-        rate: playbackRate,
-        onload: handleLoad,
-        ...delegated,
+    let isCancelled:boolean = false;
+    (() => {
+      import('howler').then(mod => {
+        if (!isCancelled) {
+          HowlConstructor.current = mod.Howl;
+
+          const sound = new HowlConstructor.current({
+            src: [url],
+            volume,
+            rate: playbackRate,
+            onload: handleLoad,
+            ...delegated,
+          });
+
+          setSound(sound);
+        }
       });
-
-      setSound(sound);
-    });
+    })();
```

Comment:
Hey @kaelhem, thanks for making the changes, and sorry for the delay.

I'm a bit confused by the IIFE 🤔 

I'm gonna merge your PR but remove it before publishing a new package. If this does have a benefit I'm not aware of, please let me know and I'll re-add it =)